src/pages/users-list-table.tsx

```diff
@@ -1,17 +1,36 @@
 import React from 'react';
+import { useSelector } from 'react-redux';
 import Table from '@material-ui/core/Table';
 import TableContainer from '@material-ui/core/TableContainer';
 import TableHeader from '../components/users-list-table-header';
 import TableBody from '../components/users-list-table-body';
+import Loader from '../components/loader';
+import Notification from '../components/notifications';
+import { columns } from '../utils';
+import { IState } from '../store';
 
 interface IProps {
-  handleFormVisibility: (isFormVisible: boolean) => void
+  handleFormVisibility: (isFormVisible: boolean) => void,
 }
 
-export default (props: IProps) =>
-  <TableContainer>
-    <Table>
-      <TableHeader />
-      <TableBody {...props}/>
-    </Table>
-  </TableContainer>
\ No newline at end of file
+export default (props: IProps) => {
+  const isLoading = useSelector(({ users }: IState) => users.isUsersLoading);
```

Comment:
we can have in our global store object user with prop just isLoading. To give  more specific names is a good approach but  I think here we can simplify